### PR TITLE
Filtering resources plugin uses project attributes and properties

### PR DESCRIPTION
**Pull request by AlexeySanko**

### CODE REVIEW

File: documentation/plugins.md [hunk 1]
```diff
@@ -823,7 +823,7 @@ The `copy_resources` plugin will do this for you at build-time.
 ### Filtering files
 
 With the `filter_resources` plugin, it is possible to replace placeholder values of type `${version}` with actual values at build-time.
-The actual values are attributes of the `project` object, so `${version}` will be replaced with the value of `project.version`.
+The actual values are attributes and properties dict of the `project` object, so `${version}` will be replaced with the value of `project.version` and `${some_property}` by `project.get_property('some_property')`.
```
Comment (esc) [inline documentation/plugins.md]: Change to: `The actual values are attributes and the properties dict` without an article, the wording is rough.

Comment (AlexeySanko) [reply documentation/plugins.md]: `The actual values are attributes `project` object and project properties` ?

Comment (esc) [reply documentation/plugins.md]: More like:

```
The actual values are the attributes and the properties dict of the `project` object. So `${version}` will be replaced with the value of `project.version` and `${some_property}` by `project.get_property('some_property')`.
```

Mainly, the article `the` makes the paragraph read more fluently, makes sense?

Comment (AlexeySanko) [reply documentation/plugins.md]: @esc thanks. Fixed

**Pull request merged**